src/models.rs

```diff
@@ -583,7 +583,10 @@ impl From<ExponentialDecayLengthPenalty> for pb::fmaas::decoding_parameters::Len
 impl From<GuardrailsTextGenerationParameters> for pb::fmaas::Parameters {
     // NOTE: types should really be consistent between APIs
     fn from(value: GuardrailsTextGenerationParameters) -> Self {
-        let decoding_method = value.decoding_method.unwrap_or("GREEDY".to_string());
+        let mut decoding_method = value.decoding_method.unwrap_or("GREEDY".to_string());
+        if decoding_method == "SAMPLING" {
+            decoding_method = String::from("SAMPLE");
+        }
```

Comment:
I am yet to create the unit test for this.
Is there a _rustier_ way of addressing this, though?

Reply:
hmm. actually why is there `SAMPLING` vs `SAMPLE` discrepancy. We should just use the name used by TGIS 🤔

Reply:
This seems to be the only place in the code that uses it. Let's close this issue and make sure we call the API using "SAMPLE" instead of "SAMPLING".